client/payment-details/summary/missing-order-notice/index.tsx

```diff
@@ -0,0 +1,145 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { Button, RadioControl } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import Loadable from 'wcpay/components/loadable';
+import { useState } from '@wordpress/element';
+import ConfirmationModal from 'wcpay/components/confirmation-modal';
+import interpolateComponents from '@automattic/interpolate-components';
+
+/**
+ * Internal dependencies.
+ */
+
+import './style.scss';
+import CardNotice from 'wcpay/components/card-notice';
+
+interface MissingOrderNoticeProps {
+	isLoading: boolean;
+	formattedAmount: string;
+}
+
+const MissingOrderNotice: React.FC< MissingOrderNoticeProps > = ( {
+	isLoading,
+	formattedAmount,
+} ) => {
+	const [ isModalOpen, setIsModalOpen ] = useState( false );
+
+	const [ reason, setReason ] = useState< string | null >( null );
+
+	const handleOnButtonClick = () => {
+		setIsModalOpen( true );
+	};
+
+	const handleModalCancel = () => {
+		setIsModalOpen( false );
+	};
+
+	const handleModalConfirmation = () => {
+		// TODO: Handle the refund.
+	};
+
+	return (
+		<>
+			<Loadable isLoading={ isLoading } placeholder="">
+				<CardNotice
+					actions={
+						<Button
+							isPrimary={ true }
```

Comment:
VSCode shows a notice saying `isPrimary` prop is marked as deprecated.

<img width="513" alt="Screenshot 2023-10-20 at 19 53 33" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/732338af-ea61-415c-9af7-dd2183dfc727">

Can we use the suggested approach `variant="primary"` instead of `isPrimary`?

```suggestion
							variant="primary"
```